lib/src/meta.dart

```diff
@@ -99,3 +99,66 @@ void _forEachGeomInGeometryObject(
     throw Exception('Unknown Geometry Type');
   }
 }
+
+/// Callback for propEach
+typedef PropEachCallback = dynamic Function(
+    Map<String, dynamic> currentProperties, num featureIndex);
```

Comment:
see other comment, I think `currentProperties` should be null. In the .JS implementation this could have been undefined the way that code was written.